features/step_definitions/visitor_can_register_for_an_account.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Visitor can register for an account
+
+    As a Visitor,
+    In order to access unique features on the news app,
+    I would like to be able to register for an account on the site.
+
+    Background:
+        Given the following user exists
+        | name | email          | password | password_confirmation |
+        | Bill | bill@email.com | password | password              |
+        Given I visit the landing page
+        And I click 'Register'
+
+    Scenario: When I fill in the sign up form correctly, I can sign up
+        Given I fill in 'Email' field with 'mail@email.com'
+        And I fill in 'Password' field with 'password'
```

Comment:
you have added a name field to the registration, why dont you test that one here?

Reply:
Name field is removed since we do not need it at this point